sqladmin/application.py

```diff
@@ -493,3 +518,28 @@ def wrap(func):
         return func
 
     return wrap
+
+
+def action(
```

Comment:
Maybe this should go to `sqladmin.actions`.

Reply:
you mean move it to new file ?